packages/ui/src/box/types.ts

```diff
@@ -70,20 +71,8 @@ export interface TextTransform {
     | CSS.TextTransformProperty;
 }
 
-export type TextStyleValues =
-  | 'display.large'
-  | 'display.small'
-  | 'body.large.medium'
-  | 'body.large'
-  | 'body.small.medium'
-  | 'body.small'
-  | 'caption'
-  | 'caption.medium';
-
 export interface TextStyle {
-  textStyle?:
-    | StyledSystem.ResponsiveValue<TextStyleValues>
-    | StyledSystem.TextStyleProps['textStyle'];
+  textStyle?: TextStylesLiteral | TextStylesLiteral[];
```

Comment:
I think we need to keep StyledSystem responsive value to use it with the css prop / helper. Can you test this?

Reply:
Indeed, `css` helper doesn't work here. Don't we only need this for the props that correspond to a css property, though? 🤔  What else might we need to add here?

Reply:
Ahh cool, yeah— still works with the responsive helper method 👍